### ISSUE TITLE
iron-meta accessibility issue

### ISSUE THREAD

**Issue by lpalmaro** [PolymerElements/iron-meta#4]
I'm currently not able to access the text using a screen reader (VoiceOver on Mac). Is this something that I should be able to access? I'm not totally sure of the purpose of this element, so figured I would simply ask.

**Comment by zacharytamas** [comment]
This element is used to share data between Polymer components on a page without having to explicitly pass around a reference to the data. There isn't a visual interface for it and the data it stores is strictly for the application so I'm not sure that it needs to be accessible.

**Comment by lpalmaro** [comment]
Okay, that makes sense. Should be good to close this out then.

**Comment by zacharytamas** [comment]
I'm not on the Polymer team, though, I've just been involved with Polymer for about a year. I could be wrong about this component so maybe a Polymer team member will chime in here.
